### PR TITLE
CB-13424: "cordova plugin add/remove" writes changes in config.xml

### PR DESCRIPTION
**Pull request by JoseExposito**
See https://issues.apache.org/jira/browse/CB-13424 and https://github.com/apache/cordova-plugin-camera/pull/290 for a description of the issue.

This PR allows to add/remove plugins to the config.xml.

### CODE REVIEW

**Comment by janpio** [comment]
Yay, that was quick. Can you talk me (and everybody reading) through the changes? Where does the logic come from? ios?

**Comment by JoseExposito** [comment]
Indeed, the logic comes from iOS/Android.

Basically, I updated the cordova-common library to match the iOS version and used it in the plugin add/remove hooks like other platforms are doing.

In my opinion the macOS platform will need more updates to match the structure other platforms are using... But for the moment this fixes the issue. Once I see if the team is open to community PRs you can expect more changes :)

File: bin/templates/scripts/cordova/Api.js [hunk 1]
```diff
@@ -199,8 +201,8 @@ Api.prototype.addPlugin = function (plugin, installOptions) {
  *   CordovaError instance.
  */
 Api.prototype.removePlugin = function (plugin, uninstallOptions) {
-    var Plugman = require('./lib/plugman/Plugman');
-    return Plugman.get(this.locations, this.events).removePlugin(plugin, uninstallOptions);
+    var xcodeproj = projectFile.parse(this.locations);
+    return PluginManager.get(self.platform, self.locations, xcodeproj).removePlugin(plugin, uninstallOptions)
```
Comment (giuseppelt) [inline bin/templates/scripts/cordova/Api.js]: `self` is undefined here
replace with `this`

**Comment by JoseExposito** [comment]
Thanks for reviewing @giuseppelt, changed

**Comment by stevengill** [comment]
I merged this in, but testing with https://github.com/apache/cordova-plugin-camera/pull/290, but it still doesn't add the feature tag to config.xml.

**Pull request merged**